t/t2350-resource-list.t

```diff
@@ -541,6 +541,32 @@ test_expect_success 'flux resource list --queue works for a queue with no constr
 	test $(grep "allocated 1" queue-every.out | grep -c debug) -eq 1 &&
 	test $(grep "allocated 1" queue-every.out | grep -c every) -eq 2
 '
+test_expect_success 'flux resource list includes queue names for empty sets' '
+	# There are no nodes in 'down' state in this test, so use that:
+	flux resource list -s down -no "{queue} {state} {nnodes}" \
+		>queue-empty1.out &&
+	cat <<-EOF >queue-empty1.expected &&
+	every down 0
+	batch down 0
+	debug down 0
+	EOF
+	test_cmp queue-empty1.expected queue-empty1.out
+'
+test_expect_success 'flux resource list includes queue names for empty sets' '
```

Comment:
i think test name same as the prior one, differentiate it in some way?  and maybe tweak the following one depending on how you change it.